experimental/private-methods.md

```diff
@@ -0,0 +1,22 @@
+[Private class instance methods, getters and setters](https://github.com/tc39/proposal-private-methods)
+
+```js
+interface PrivateName <: Node {
+    type: "PrivateName";
+    name: string;
+}
+
+extend interface MethodDefinition {
+    key: Expression | PrivateName;
+}
+```
+
+Private methods cannot be static or have computed names.
```

Comment:
Per https://github.com/tc39/proposal-static-class-features/, private methods can be static.